client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -208,7 +207,7 @@ endif()
 # NOTE: Platform specific compilation flags should be set 
 # in the corresponding platform CMake script(s).
 if (NOT MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+    list(APPEND KAA_COMPILE_OPTIONS -fPIC)
```

Comment:
By the way, I am going to remove fPIC because providing only single type library (static|shared)